pipeline_templates/disable_appverifier.yml

```diff
@@ -0,0 +1,14 @@
+# Convenience helper to disable application verifier for all processes
+# May be useful as a cleanup task in case the application verifier state leaks from other canceled jobs
+
+parameters:
+ - name: repo_root
+   default: $(Build.SourcesDirectory)/deps/c-build-tools
+
+steps:
+  - task: PowerShell@2
+    displayName: 'Disable AppVerifier'
+    inputs:
+      targetType: 'filePath'
+      filePath: '${{ parameters.repo_root }}/pipeline_templates/scripts/enable_appverifier_ctest_tests.ps1'

```

Comment:
>enable_appverifier_ctest_tests [](http://example.com/codeflow?start=72&length=30)

this says "enable", but line 10 is "disable" and the comments at top says "disable".

Reply:
Maybe it should have a parameter (on) set to false?

Reply:
renamed the script, "-on" is a flag and without it it will disable app verifier